recipes/cysgp4/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cysgp4" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "24bf9cdc2168c7fc6b6dd89a24849c09560b755d7268e97f6988c08aa09ff84e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - openmp 4.0.*  # [osx]
+    - python
+    - pip
+    - setuptools
+    - setuptools_scm
+    - cython
+    - numpy
+
+  run:
+    - python
+    - openmp 4.0.*  # [osx]
+    - {{ pin_compatible('numpy') }}
+
+test:
+  requires:
+    - setuptools
```

Comment:
`setuptools` should go into `requirements/run` since `cysgp4.version` uses `pkg_resources` which is part of `setuptools`. When you added this to `requirements/run`, you can remove it from `test/requires`.